### ISSUE TITLE
[FEATURE] calculate statistics on training prediction/plan

### ISSUE THREAD

**Issue by wsbrenk** [akasolace/HO#750]
![trainingstatistics](https://user-images.githubusercontent.com/11821502/97108336-292f3b00-16cd-11eb-9dc1-0ee259d059e2.png)

i would like to have an overview/aggregation of the training plan of the whole team

**Comment by akasolace** [comment]
maybe this can be put in a new tab ? I am worry the exisiting tab would be too bloated if you had this new table into it.  What do you think ?

**Comment by wsbrenk** [comment]
good idea - i thought the player details are somewhat to big, but this place could be used to display the training history and future plans beneath the player training table, which could be prolonged to the complete screen width.
